mixer/pkg/adapter/instanceUtil.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adapter
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"time"
+)
+
+// This file contains common utility functions that adapters need to process content
+// of instances.
+
+// Stringify converts basic data types, supported by `istio.mixer.v1.config.descriptor.ValueType`, into string.
+// Note nil object is converted into empty "" string.
+func Stringify(v interface{}) string {
+	if v != nil {
+		// Switch on all possible ValueTypes
+		switch vv := v.(type) {
+		case string:
+			return vv
+		case int64:
+			return strconv.FormatInt(vv, 10)
+		case float64:
+			return strconv.FormatFloat(vv, 'f', -1, 64)
+		case bool:
+			return strconv.FormatBool(vv)
+		case time.Time:
+			return vv.String()
```

Comment:
This is actually quite slow, and produces output with spaces which I think would be best to avoid. Could you lift the time formatting code from pkg/log for this?

Reply:
Fair point. followup question : 
So, when operator is writing overriding dimensions for a label of type `time.Time`, how would they write a string version of it ? Inside adapter config, the adapter developer should explicitly mention what string format of time should be written by operator when writing override dimensions, and therefore adapter developer should exactly know how are we stringifying time.Time in this function. Therefore, I was thinking using default `String()` would help keep both Operator (writing override label value) and adapter dev(documenting operator config for `dimensions` field) on the same page.

If we use code from pkg/log, will operator be able to write string form of  `time.Time` that we are generating there ?